lib/ui/widget/address_add_widget.dart

```diff
@@ -132,6 +134,25 @@ class AddressAddWidget extends HookWidget {
                                 color: Color.fromRGBO(75, 124, 211, 1)),
                           )
                         ]))))),
+            const SizedBox(height: 16),
+            if (chainId == eos)
+              Text.rich(TextSpan(
+                  style: TextStyle(
+                    color: context.theme.secondaryText,
+                    fontSize: 13,
+                    height: 2,
+                  ),
+                  children: [
+                    TextSpan(text: '${context.l10n.addAddressNotSupportTip} '),
+                    TextSpan(
+                        text: context.l10n.eosContractAddress,
+                        style: TextStyle(
+                          color: context.theme.text,
+                          fontWeight: FontWeight.bold,
+                        ))
+                  ]))
+            else
+              const SizedBox(),
```

Comment:
this else statement seems can be removed.

Reply:
removed